web/js/components/feature-alert/alert.js

```diff
@@ -14,9 +14,7 @@ class FeaturedAlert extends React.Component {
   }
 
   dismissAlert() {
-    if (util.browser.localStorage) {
-      localStorage.setItem('<alert-name>', true);
-    }
+    // safeLocalStorage.setItem('<alert-name-goes-here>', true);
```

Comment:
Do we want to `setItem` here? Also, I'm not sure that this file is still in use (Maybe it is still be useful for temporary alerts?).

Reply:
I kept it around for re-use in the event that we want to display another temporary "new feature" alert.  Currently it's not in use though.